### PR TITLE
Realistic 3 d

### PR DESCRIPTION
**Pull request by mloubout**
Correction of the PDE for TTI
Fixed the substituted values in TTI 
Marmousi TTI example to be added

### CODE REVIEW

File: examples/tti_operators.py [hunk 1]
```diff
@@ -151,21 +151,26 @@ def Bhaskaracos(angle):
...
+            Gyyp = ang1**2 * u.dx2 + ang2**2 * u.dy2 - (2 * ang3 * ang2)**2 * u.dxy
+            Gzzr = ang2**2 * ang1**2 * v.dx2 + ang3**2 * ang1**2 * v.dy2 + ang0**2 * v.dz2 + 2 * ang3 * ang2 * ang1**2 * v.dxy + ang3 * 2 * ang1 * ang0 * v.dyz + ang2 * 2 * ang1 * ang0 * v.dxz
+        else:
+            Gyyp = 0
+            Gxxp = ang0**2 * u.dx2 + ang1**2 * u.dz2 - 2 * ang0 * ang1 * u.dxz
+            Gzzr = ang1**2 * v.dx2 + ang0**2 * v.dz2 + 2 * ang0 * ang1 * v.dxz
         # Derive stencil from symbolic equation
         stencilp = 2 * s**2 / (2 * m + s * damp) * (2 * m / s**2 * u + (s * damp - 2 * m) / (2 * s**2) * u.backward + A * (Gxxp + Gyyp) + B * Gzzr)
-        stencilp = factor(expand(stencilp))
+        # stencilp = factor(expand(stencilp))
```
Comment (mlange05) [inline examples/tti_operators.py]: Can we remove these commented out lines if they are not used?

Comment (mloubout) [inline examples/tti_operators.py]: I left it there to remember the stencil is not factorized at all and that there is something to do there but I can remove it.

**Comment by mlange05** [comment]
I don't fully understand the details of the TTI runs, but from my point of view this looks fine. A second pair of eyes might be good, but otherwise I think this is good to go once travis is happy.

**Comment by mloubout** [comment]
What do you mean by the details of TTI run?

File: examples/tti_operators.py (hunk 1 above)
Comment (mlange05) [inline examples/tti_operators.py]: Ok, if it is still useful, I'm ok with leaving it in.

**Comment by mlange05** [comment]
Oh, I just mean I can't judge any of the spacing, time-stepping or coefficient changes, but I trust you are right. :wink:

**Pull request merged**